src/core_modules/capture-core/components/Pages/EnrollmentAddEvent/ProgramStageSelector/ProgramStageSelector.component.js

```diff
@@ -40,18 +32,29 @@ const ProgramStageSelectorComponentPlain = ({ programStages, onSelectProgramStag
                 >
                     <Button
                         className={classes.button}
-                        big
                         secondary
                         disabled={disableStage}
                         onClick={() => onSelectProgramStage(programStage.id)}
                         dataTest={'program-stage-selector-button'}
+                        icon={
+                            programStage.style?.icon && (
+                                <div className={classes.icon}>
+                                    <NonBundledDhis2Icon
+                                        name={programStage.style?.icon}
+                                        color={programStage.style?.color}
+                                        width={24}
+                                        height={24}
+                                        cornerRadius={5}
+                                    />
+                                </div>
+                            )
+                        }
```

Comment:
Checks to see if an icon exists for the program and omits if not. Previously, there was a "default" program icon shown that I don't think was helpful.

Also moved to use the `Button` `icon` property.